guides/security/BYOD.md

```diff
@@ -0,0 +1,101 @@
+- Web browsers
+- Extensions
+- Plug-ins
+
+You must update your software within 14 days of a new verison being released.
+
+### Use endpoint protection software
+Unless your device is operating iOS or ChromeOS, your device must have endpoint protection software installed, which must also be kept up to date. 
+
+Endpoint protection is more than just antivirus software, and usually includes a firewall and may have other tools to protect your device, like network threat detection.
+
+Any device you plug into the device, like an external USB device, must also be scanned for viruses. We recommend doing a full scan at least once a month.
+
+### Set a password/passphrase
+Your device must require a password/passphrase to login, return from screensaver or wake from sleep. Ideally your device will lock the screen (or start the screensaver) after a few minutes (no more than 5) of the device being idle
+
+### How you manage passwords/passphrases
+Your password/passphrase must conform to the Made Tech Password Policy which will be linked below. You should also use a password manager and 2FA.
+
+### How you connect your device to networks
+We have our own VPN which automatically authenticates with G Suite. Everyone can use the service (not only engineers) for a more secure experience (i.e. protecting yourself when using a public WiFi). To connect to the VPN, please refer to the [guide found in the Handbook.](https://github.com/madetech/handbook/tree/master/guides/vpn) 
+
+### How you store & manage data
+If data is stored and managed incorrectly it can be lost; stolen or changed (including someone loading a virus). ‘Storing data’ includes on your computer (whether it is on or off) and portable media, like USB sticks. Data should be encrypted wherever possible.
+
+If you don’t need files and data any more, delete them and make sure they can’t be recovered.
+
+### Disk Encryption
+You must encrypt all your data storage devices. This includes:
+
+ - hard disks (including on smartphones and tablets)
+ - cloud storage
+ - USB sticks
+
+If these aren’t encrypted, people can see any data that’s on them or install viruses without you noticing
+
+### Turn your device off, rather than putting it to sleep
+Depending on the device, data may only be encrypted when the device itself is turned off. So please be aware that closing the lid to put the device to sleep won’t necessarily enable encryption; in which case the data will not be protected.
+
+If your device will not be used for a significant period of time, we suggest turning it off entirely to ensure disk encryption is enabled. Most modern operating systems will allow you to resume all the files/applications that were open before shutdown.
+
+Turning off your device to ensure disk encryption is enabled is important if your device will be unattended. This could include when inside a backpack or carrying case when travelling, as those can be lost or stolen
+
+### Store data in Made Tech tools
```

Comment:
That said this can probably be simplified to something along the lines of "Store data in G-Suite or GitHub. Do not store data locally on your device"?

Reply:
G-Suite or GitHub
- doesn't cover where all data is stored (Harvest/Xero/HubSpot/wherever the visitor ipad backs up to)

Reply:
As far as I can think G-Suite and GitHub are the only tools where you're likely to be working on a local copy of anything while needing to sync it back? And only G-Suite in quite extreme examples given Google Docs et al.

I think the ultimate guideline here is do not store data locally on your device, ensure it's synced to the relevant cloud service. Do we actually need to say any more than that?

Reply:
amended, but have left the line about 

> If you are working offline and cannot access the cloud service